api/v1alpha1/contour_types.go

```diff
@@ -91,6 +91,14 @@ type ContourSpec struct {
 	// +optional
 	GatewayClassRef *string `json:"gatewayClassRef,omitempty"`
 
+	// GatewayControllerName is used to determine which GatewayClass
```

Comment:
This will be a same value with https://github.com/projectcontour/contour/blob/e41d550bf3a96fe7c5035f3dc4cfb3f4967329c4/pkg/config/parameters.go#L366-L369 ?
If so, I think that `The string takes the form of "projectcontour.io/<namespace>/contour".` should be mentioned here too.